ImagePicker/ImagePickerAssetModel.swift

```diff
@@ -1,89 +1,67 @@
-//
-//  ImagePickerAssetModel.swift
-//  ImagePicker
-//
-//  Created by Peter Stajger on 12/09/2017.
-//  Copyright © 2017 Inloop. All rights reserved.
-//
+// Copyright © 2018 INLOOPX. All rights reserved.
 
-import Foundation
 import Photos
 
-///
 /// Model that is used when accessing an caching PHAsset objects
-///
+
 final class ImagePickerAssetModel {
-    
-    deinit {
-        log("deinit: \(String(describing: self))")
-    }
-    
     var fetchResult: PHFetchResult<PHAsset>! {
         set { userDefinedFetchResult = newValue }
         get { return userDefinedFetchResult ?? defaultFetchResult }
     }
     
+    // Will be use for caching
+    var previousPreheatRect = CGRect.zero
```

Comment:
if the class is called model, the properties should be private and unmodifiable from outside

Reply:
Class renamed